[Doc] Show how to create repository in Aliyun OSS (#30096)
StarRocks/starrocks: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md

> StarRocks supports creating repositories in AWS S3 only according to the S3A protocol. Therefore, when you create repositories in AWS S3, you must replace `s3://` in the S3 URI you pass as a repository location in `ON LOCATION` with `s3a://`.

- For Google GCS:
  - "fs.s3a.access.key": The Access Key used to access the Google GCS bucket.
  - "fs.s3a.secret.key": The Secret Key used to access the Google GCS bucket.
  - "fs.s3a.endpoint": The endpoint used to access the Google GCS bucket.

> **NOTE**
>
> StarRocks supports creating repositories in Google GCS only according to the S3A protocol. Therefore, when you create repositories in Google GCS, you must replace the prefix in the GCS URI you pass as a repository location in `ON LOCATION` with `s3a://`.

> - For Aliyun OSS:

Reviewer: there are two ways to access oss, 1) fs.oss.xxx which uses the jindo sdk 2) aws.s3.xxx, which uses aws s3 sdk. we may need to mention both? if jindosdk is not working in some cases.

>
> StarRocks supports creating repositories in Google GCS only according to the S3A protocol. Therefore, when you create repositories in Google GCS, you must replace the prefix in the GCS URI you pass as a repository location in `ON LOCATION` with `s3a://`.

- For Aliyun OSS:

>   - "fs.oss.accessKeyId": The Access Key ID that you can use to access the Aliyun OSS bucket. 

Reviewer 2: ```suggestion
  - "fs.oss.accessKeyId": The Access Key ID used to access the Aliyun OSS bucket. 
```

>   - "fs.oss.accessKeySecret": The Secret Access Key that you can use to access the Aliyun OSS bucket.

Reviewer 2: ```suggestion
  - "fs.oss.accessKeySecret": The Secret Access Key used to access the Aliyun OSS bucket.
```

>   - "fs.oss.endpoint": The endpoint that you can use to access the Aliyun OSS bucket.

Reviewer 2: ```suggestion
  - "fs.oss.endpoint": The endpoint used to access the Aliyun OSS bucket.
```

A little shorter

Author: Thank you for your suggestion. I have updated the wording here based on your feedback, and also made similar adjustments in other places with the same issue.
